.firebaserc

```diff
@@ -1,5 +1,5 @@
 {
   "projects": {
-    "default": "cvoxel-testnet"
+    "default": "cvoxel-dev"
```

Comment:
[NOTE] Changed because I could not connect to firestore when using the emulator locally